yarn.lock

```diff
@@ -6975,7 +6980,25 @@ copy-descriptor@^0.1.0:
   resolved "https://registry.yarnpkg.com/copy-descriptor/-/copy-descriptor-0.1.1.tgz#676f6eb3c39997c2ee1ac3a924fd6124748f578d"
   integrity sha1-Z29us8OZl8LuGsOpJP1hJHSPV40=
 
-copy-webpack-plugin@^5.0.2, copy-webpack-plugin@^5.1.1:
+copy-webpack-plugin-v5@^5.1.2:
+  version "5.1.2"
+  resolved "https://registry.yarnpkg.com/copy-webpack-plugin-v5/-/copy-webpack-plugin-v5-5.1.2.tgz#e7fd3e55158f636abaf91130505a81a41c7eed01"
+  integrity sha512-kTX5+l1ApkXR98hbJDGnU1yILZ7zpRdBN1rxE7xq40u9JFNOtgf+aEBqUebtFlGiSE0qecf6xbB8bI9Ys0YByQ==
+  dependencies:
+    cacache "^12.0.3"
+    find-cache-dir "^2.1.0"
+    glob-parent "^3.1.0"
+    globby "^7.1.1"
+    is-glob "^4.0.1"
+    loader-utils "^1.2.3"
+    minimatch "^3.0.4"
+    normalize-path "^3.0.0"
+    p-limit "^2.2.1"
+    schema-utils "^1.0.0"
+    serialize-javascript "^4.0.0"
+    webpack-log "^2.0.0"
+
+copy-webpack-plugin@^5.0.2:
```

Comment:
Thanks for the pr.
Do you know why there is a second copy-webpack-plugin here?

Reply:
It comes from vuepress, which is used to deploy the documentation.